src/actions/dataset.ts

```diff
@@ -13,6 +13,15 @@ import {SHELF_CLEAR} from './shelf';
 
 import {getConfig} from '../selectors';
 
+import {Type} from 'vega-lite/src/type';
+
+export type FieldAction = ChangeFieldType;
+export const CHANGE_FIELD_TYPE = 'CHANGE_FIELD_TYPE';
```

Comment:
Let's name this `DATASET_SCHEME_CHANGE_FIELD_TYPE`.
(If you notice my naming pattern, I use the state part as prefix for actions.)

cc: @starry97 -- this would be good for you to know too.